app/src/main/res/values/strings.xml

```diff
@@ -18,4 +18,27 @@
 
     <!-- Placeholder text for unimplemented screens -->
     <string name="hello_blank_fragment">TODO - Not Implemented</string>
+    <string name="app_name">course-project-tindar</string>
```

Comment:
There are two <string name ="app_name"> (line 2 and line 21). One has to be deleted.